gnd/src/main/java/com/google/android/gnd/ui/map/gms/GoogleMapsMapAdapter.java

```diff
@@ -102,8 +101,13 @@ public GoogleMapsMapAdapter(GoogleMap map, Context context) {
 
   public void renderJsonLayer() {
     File file = new File(context.getFilesDir(), GEO_JSON_FILE);
-    if (file.exists()) {
-      try {
+
+    if (!file.exists()) {
+      Log.e(TAG, "GeoJSON file not found.");
+      return;
+    }
```

Comment:
This is redundant since FileNotFoundException is an IOException that will get caught below.

Reply:
Thanks!